test/uk/gov/hmrc/statepension/domain/nps/StatePensionSpec.scala

```diff
@@ -138,6 +140,16 @@ class StatePensionSpec extends StatePensionUnitSpec {
     }
   }
 
+  "Abroad customer" should {
```

Comment:
Can you use the term abroad/overseas auto credits or something, it's very confusing as there is logic in the service for just abroad people, which has nothing to do with the exclusion.